admin/bower.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "OSF_admin",
+  "dependencies": {
+    "jquery": "^1.11.2",
```

Comment:
I'd agree since I couldn't seem to find any tooling to have bower actually install these packages. I had to run bower install by hand and then run `bower install`. Running the webpack config didn't seem to run bower while I believe the inv assets command in osf proper already installs all these things for you.